pkg/transcriptionsystems/bipa/consonants.tsv

```diff
@@ -606,3 +609,10 @@ kʲ	voiceless	velar	stop		palatalization:palatalized
 k̚	voiceless	velar	stop		release:unreleased	
 kʷʼ	voiceless	velar	stop	+	labialization:labialized,ejection:ejective	
 ɡ̤	voiced	velar	stop	+	breathiness:breathy	
+d̪l̪	voiced	lateral	affricate			
+ɽr	voiced	retroflex	trill			
+kp	voiceless	labio-velar	stop			
+gb	voiced	labio-velar	stop			
+qʡ	voiceless	uvular-epiglottal	stop			
+nm	voiced	labio-alveolar	nasal			
+ŋm	voiced	labio-velar	nasal			
```

Comment:
same here, I do not see the point for this.

Reply:
It is an IPA sound that is used in Phoible inventories: https://phoible.org/parameters/9D6EBFB925E507CEC3D4A539BFD0BF08#3/-4.25/81.07

Reply:
It is a cluster consonant as well. The problem is that we do not allow clusters for the same sound type so far (nasal + nasal), as this would be too powerful. So the best is then to fix the restriction in the code, that it can be accepted as a cluster.

Reply:
This one I would defend, as parallel to /kp/, /gb/. I don't see a principled reason to exclude a "cluster" of nasal stops but not of oral stops. In fact, they are very much parallel, see also https://phoible.org/parameters/8EB42EA410A32B30095194B613991F57#2/3.1/65.4, which by the same logic would also warrant inclusion.